flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/internal/BaseExpressions.java

```diff
@@ -722,6 +723,23 @@ public OutType round(InType places) {
         return toApiSpecificExpression(unresolvedCall(ROUND, toExpr(), objectToExpression(places)));
     }
 
+    /**
+     * Converts numbers between different number bases.
+     *
+     * <p>Returns a string representation of the number N, converted from base `fromBase` to base
+     * `toBase`. Returns NULL if any argument is NULL. The argument N is interpreted as an integer,
+     * but may be specified as an integer or a string. The minimum base is 2 and the maximum base is
+     * 36. If fromBase is a negative number, N is regarded as a signed number. Otherwise, N is
+     * treated as unsigned. CONV() works with 64-bit precision.
+     *
+     * <p>E.g. 'a'.conv(16, 2) leads to '1010', 100.conv(10, -8) leads to "144".
+     */
+    public OutType conv(InType fromBase, InType toBase) {
```

Comment:
Could you also add the corresponding Python API? See https://github.com/apache/flink/blob/master/flink-python/pyflink/table/expression.py for for more details.

Reply:
it exists in this commits